docs/upgrade-notes/serverless-v2-upgrade.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
These instructions were hammered out in https://bugs.earthdata.nasa.gov/browse/CUMULUS-3670